Makefile

```diff
@@ -39,7 +39,14 @@ train-docker-gpu:
 	docker run -it --rm --user $(id -u):$(id -g) --env-file .env -v $(shell pwd):/usr/src/app --gpus all train:latest
```

Comment:
can you edit this one too? is the same as the other one plus `--gpus all`

Reply:
I added a param to the command so you can specify it